### ISSUE TITLE
Identifisering

### ISSUE THREAD

**Issue by anjamaa** [realfagstermer/realfagstermer#988]
Er den norske Wiki-artikkelen litt "feil"? Den handler om identifikasjon av mennesker, mens den engelske handler om "artsbestemmelse" (i mangel på bedre ord)?

--
URI: http://data.ub.uio.no/realfagstermer/c012836
Bruk: http://ub-viz01.uio.no/okapi2/#/search?q=realfagstermer%3A%22Identifisering%22
Soksed: https://soksed.biblionaut.net/concepts/REAL012836

**Comment by Hypsibius** [comment]
Oooops!

**Comment by Hypsibius** [comment]
Spesifisere? Artsidentifikasjon? Identifisering (arter)?

**Comment by danmichaelo** [comment]
Ja, der var det en feil kobling. Har fiksa. Ser ikke ut som det er noen artikkel på norsk om emnet.

Brukes forresten `Bestemmelse` som synonym for `Identifisering` eller er det forskjellige ting?

**Comment by Hypsibius** [comment]
Vi snakkar gjerne om 'artsbestemmelse' i den samanheng

**Comment by BioHeidi** [comment]
Jeg er fristet til å slette termen Identifisering. Eller så må vi katalogrydde over på Artsbestemmelse/Bestemmelseslitteratur, Legemiddelanalyse, Mikrobiologiske metoder, Analytisk kjemi, Genetisk kartlegging etc etc

**Comment by Caroline-A** [comment]
Jeg så du kommentert på den @BioHeidi og tenkt å spørre - når skal vi bruke artsbestemmelse vs. bestemmelseslitteratur? Jeg er litt usikker på forskjellen i bruk hos oss.

**Comment by Hypsibius** [comment]
Hm, godt spørsmål, @Caroline-A : tenkjer at bestemmelseslitteratur er nøklar med ganske instrumentell tilnærming, medan artsbestemmelse kan handle om sams metodikk, alt frå mikroskopering og bar-coding til artsbegrep etc? Or not?

**Comment by Caroline-A** [comment]
Jeg synes det er en god tanke @Hypsibius , jeg ser forskjellen der! Kanskje det kunne ha vært fint å legge til en DE til termene for å sikre for at vi bruker dem på samme måte?

**Comment by grosyn** [comment]
I samband med rydding i formene våre, lurer eg på om det er ok at eg i dei postane som inneheld både 655 Floraer og 655 Bestemmelseslitteratur, kan sletta Bestemmelseslitteratur? Det same vil gjelda for Soppfloraer vs Bestemmelseslitteratur.

@BioHeidi
